### PR TITLE
Update torch example to match LUMI after the update

### PR DESCRIPTION
**Pull request by TheBlackKoala**

### CODE REVIEW

**Review by joasode — APPROVED** [review]
It looks good.

File: examples/LUMI/conda_pytorch_rocm/README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ On LUMI, the container may be built using:
 ```bash
 module load LUMI
```
Comment (joasode) [inline examples/LUMI/conda_pytorch_rocm/README.md:10]:
```suggestion
module load CrayEnv
```

File: examples/LUMI/conda_pytorch_rocm/README.md
Comment (joasode) [inline examples/LUMI/conda_pytorch_rocm/README.md:1]: The change to `minimal_pytorch.yml` should be reflected in the text in the link to `https://download.pytorch.org/whl/rocm5.4.2/` to `https://download.pytorch.org/whl/rocm6.0.3/`

**Review by joasode — CHANGES_REQUESTED** [review]

File: examples/LUMI/conda_pytorch_rocm/README.md
Comment (TheBlackKoala) [reply examples/LUMI/conda_pytorch_rocm/README.md:1]: The version should be removed - which it has been in the example but not in the readme - updated readme

**Review by joasode — APPROVED** [review]

**Pull request merged**